app/controllers/sac_cas/event/participations_controller.rb

```diff
@@ -38,6 +38,14 @@ def new
 
   private
 
+  def permitted_attrs
+    permitted = self.class.permitted_attrs.dup
```

Comment:
Vielleicht kann dies mit Zeile 16 zusammengeführt werden, damit es etwas aufgeräumter ist?

Reply:
Hmm... ich mag eigentlich dass es aufgeteilt ist.
Alle Attribute welche einfach fix den `permitted_attrs` hinzugefügt werden, werden oben definiert und alles was dann noch dynamisch entschieden wird in der Methode.
So haben wir das beispielsweise schon bei der PBS gemacht: https://github.com/hitobito/hitobito_pbs/blob/8aa963c19b0bb8090d6850bd290c081b08538985/app/controllers/pbs/event/participations_controller.rb#L75

Andererseits kann es dann passieren, dass man eines der beiden übersieht wenn man es später mal debugged.. Ich tendiere aber dennoch es so zu lassen :thinking:

Reply:
Alles klar :) Dann lassen wir es.